ckanext/qgov/common/anti_csrf.py

```diff
@@ -131,11 +176,11 @@ def is_request_exempt():
     return not is_logged_in() or API_URL.match(request.path) or request.method in {'GET', 'HEAD', 'OPTIONS'}
 
 def anti_csrf_before(obj, action, **params):
+    RAW_BEFORE(obj, action)
```

Comment:
why was this moved?

Reply:
The built-in `before` function initialises some necessary objects such as the current user.